rviz_default_plugins/CMakeLists.txt

```diff
@@ -281,6 +281,7 @@ ament_export_dependencies(
   tf2_ros
   urdf
   visualization_msgs
+  image_transport
```

Comment:
Can you please put it in alphabetical order in the list?  Also we should do the same for the `ament_target_dependencies` above.